src/components/gds-card/gds-card.scss

```diff
@@ -3,6 +3,7 @@
 :host {
   @include card;
   transition: box-shadow 0.3s;
+  height: var(--card-height);
```

Comment:
Should this also have default value `auto`?

Reply:
@henritukiainen  Yeah, I've been thinking about what is the correct way of doing this "fallback to initial value". Is it like you suggest that we define the initial value, or that we don't define the variable and the value then default to initial value? Does it do that?

Reply:
@henritukiainen  I think in the end, setting the fallback value explicitly is probably the better option as it's clear to the developer what is going to be in the value. @dominickudiabor if you add these `auto` fallbacks and then merge?

Reply:
okay!